shared/login/forms/container.desktop.js

```diff
@@ -34,6 +34,7 @@ const styles = {
     height: '100%'
   },
   button: {
-    zIndex: 9999
+    zIndex: 9999,
+    position: 'absolute'
```

Comment:
This way we can better position the rest of the elements since we don't have to take into account the height of the back button.